From daniel.marjamaki@comhem.se Mon Nov 21 13:02:10 2005
Message-ID: <437CD62E.1000006@comhem.se>
Date: Thu, 17 Nov 2005 20:12:46 +0100
From: Daniel Marjamäki <daniel.marjamaki@comhem.se>
To: Greg KH <gregkh@suse.de>
Subject: PCI: direct.c: DBG


The DBG() call where updated with the appropriate KERN_* symbol.

Signed-off-by: Daniel Marjamäki <daniel.marjamaki@comhem.se>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 arch/i386/pci/direct.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/arch/i386/pci/direct.c
+++ gregkh-2.6/arch/i386/pci/direct.c
@@ -201,7 +201,7 @@ static int __init pci_sanity_check(struc
 			return 1;
 	}
 
-	DBG("PCI: Sanity check failed\n");
+	DBG(KERN_WARNING "PCI: Sanity check failed\n");
 	return 0;
 }
 
